<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Asiguratori</class>
 <widget class="QDialog" name="Asiguratori">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asiguratori</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/rapoarte</normaloff>:/rapoarte</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Asigurator</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tip</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>&amp;Adauga</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/add</normaloff>:/add</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>&amp;Editeaza</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/edit</normaloff>:/edit</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>&amp;Sterge</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/delete</normaloff>:/delete</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="loadingLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>A&amp;nuleaza</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/cancel</normaloff>:/cancel</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
